### PR TITLE
Feature/tradestation

### PR DESCRIPTION
**Pull request by williamtbranch**
got the getDaysBack endpoint working

### CODE REVIEW

File: NODE/src/services/TradestationService.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  let body: string = "";
+  const writable = new Writable();
+  writable._write = function(chunk, encoding, next) {
+    console.log(chunk.toString())
+    body += chunk.toString();
+    next();
+  }
+
+  const tStream = got.stream(`https://api.tradestation.com/v2/stream/barchart/SPY/1/Daily?daysBack=250&lastDate=03-12-2022`, {
```
Comment (JonahBranch) [inline NODE/src/services/TradestationService.ts:108]: replace some of these hard coded values (like Daily, daysBack and symbol) with parameters.
Also, replace all instances of `https://api.tradestation.com/v2/` with a constant defined at the top of the file somewhere. we should probably also change it to the simulated api as well

File: NODE/src/services/SecurityEndpoints.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-
 import got from 'got';
```
Comment (JonahBranch) [inline NODE/src/services/SecurityEndpoints.ts:1]: we could consider deleting this file. I don't think it's being used anywhere

File: NODE/src/index.ts [hunk 3]
```diff
@@ -15,14 +15,38 @@ ipcRenderer.on('catfact', (event, catfact: string) => {
...
-ipcRenderer.on('security:get', (event, series: Tick[]) => {
+ipcRenderer.on('quote:get', (event, quote: any) => {
+  console.log(quote); 
+})
+
+ipcRenderer.on('daily:get', (event, daily: any) => {
+  console.log(daily); 
+})
+
+ipcRenderer.on('graph', (event, data: any) => {
```
[the remark this answers is not in the document]
Comment (JonahBranch) [reply NODE/src/index.ts:26]: someday this callback is going to do something reactive

File: NODE/src/services/SecurityEndpoints.ts (hunk 2 above)
Comment (JonahBranch) [reply NODE/src/services/SecurityEndpoints.ts:1]: I mean the SecurityEndpoints.ts file, not the import got.
I couldn't figure out how to add a comment to the whole file so I just commented on the first line

Comment (williamtbranch) [reply NODE/src/services/SecurityEndpoints.ts:1]: Changed name to AVEndpoints to leave option to use alphavantage later.

**Pull request merged**